syntax_and_semantics/comparisons.md

```diff
@@ -0,0 +1,39 @@
+# Comparisons
+
+Comparing Strings
+
+```crystal
+"a" == "a"		# true : Bool
+"a" === "a"		# true : Bool
+"a" == "b"		# false : Bool
+"a" === "b"		# false : Bool
+
+"a" =~ /a/ # => 0
+"a" =~ /b/ # => nil
+
+"a" !~ /a/ # => false
+"a" !~ /b/ # => true
+
+"a" <=> "b"		# -1 : Int32
```

Comment:
I'd suggest to stick to `# => <value>` syntax for return values and drop `: <type>` altogether from comments.